twill11/hometask-6/Gemfile

```
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+source 'https://rubygems.org'
+
+git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
+
+gem 'dry-initializer'
+gem 'http'
+gem 'json'
+gem 'rubocop'
```

You should indicate versions of gems in gemfile.rb
```suggestion
gem 'dry-initializer', '~> 3.0', '>= 3.0.4'

```